### ISSUE TITLE
API Session command SetSubtitleStreamIndex does not set the subtitle

### ISSUE THREAD

**Issue by muisje** [jellyfin/jellyfin-web#6365]
**Describe The Bug**

when sending the SetSubtitleStreamIndex command with the api it does not set it to the subtitle.

**Steps To Reproduce**
1. Play a movie with subtitles
2. Find out your session id
3. Make one of these requests:
http://localhost/Sessions/{sessionId}/Command/SetSubtitleStreamIndex
http://localhost/Sessions/{sessionId}/System/SetSubtitleStreamIndex

{"Index": 0}
or 
http://localhost/Sessions/{sessionId}/Command
{"Name": "SetSubtitleStreamIndex", "ControllingUserId": {userid}, "Arguments": {"Index": 0}}

or

http://localhost/Sessions/{sessionId}/Playing

Result is that subtitle does not change or gets deactivated

**Expected Behavior**
Subtitle should change

**System (please complete the following information):**
 - Platform: Linux
 - Browser:  Firefox, Edge
 - Jellyfin Version: 
Server version
10.10.3
Web version
10.10.3
Build version
10.10.3

**Additional Context**

sending a get request to http://localhost/Sessions has in it's responses that this command is supported

It works on the android tv app

It does not work on: jellyfin media player, the android app, web browser

**Comment by muisje** [comment]
And would it possible to check in the event of a metadata update while playing it to check if the subtitle file changed? I'm asking this because I'm creating something that that generates subtitles on demand with ai transcription, it does chunks and starts when the media plays. The thing is i can't force jellyfin to use the updated subtile while the media is playing and i want to do that to use the updated subtitle.

**Comment by viown** [comment]
@muisje After trying to update the index, do you see a `Received command` message in console logs?

This seems to work on my end. Here is how I did it:

1. Send POST request to `Sessions/{sessionId}/Command`
2. In the body schema, pass the following JSON:
```json
{
	"Name": "SetSubtitleStreamIndex",
	"Arguments": {
		"Index": 2
	}
}
```

`/Sessions/{sessionId}/Command/{command}` does not seem to support passing arguments, but `Sessions/{sessionId}/Command` does.

**Comment by muisje** [comment]
@viown Thanks that works!

So would it be possible to make the session use an updated subtitle file while playing? I'm refreshing the metadata, but it seems that i manually have to start and stop playing to get the updated subtitle file

**Comment by viown** [comment]
The subtitle file has to be scanned before it can be recognized in playback. You should most likely do your processing to your media in advance.

**Comment by muisje** [comment]
Yeah i would agree, but what I'm trying to do is create the subtitle on demand. So what i mean is that the creation of the subtitle starts when a movie gets played. And while the movie is playing the subtitle file gets updated. The only thing that missing is that jellyfin picks up the new/updated subtitle file while playing. 

Why I'm trying to make this happen is that the creation of the subtitle file with ai transcribing is quite demanding and isn't quick, but it can be fast enough to do this on demand. 

So what i would like is the ability to force reloading of a subtitle file while a movie is being played if that would be possible
